tailwind.config.js

```diff
@@ -18,7 +18,7 @@ module.exports = {
     },
     extend: {
       colors: {
-        "light-background": "#F1F1F1",
+        "light-background": "#F8F8F8",
```

Comment:
Please let me know if we should make it whiter

Reply:
This looks good, but i think whiter might look better. Can you post a comparison?

Reply:
![Frame 2 (1)](https://github.com/darklang/darklang.com/assets/87861924/3e70ff4d-2bc7-479e-a416-a5775d0856a6)

Reply:
Ok probably F8 is best!

Reply:
Here are some that are closer to F8.
![Frame 3](https://github.com/darklang/darklang.com/assets/87861924/f1b031b2-413b-49b9-b2c7-8d93bd107c9c)

Reply:
FA maybe?

Reply:
I think FA looks nice! (FWIW F8 is part of our original color palette). This updates the code snippets backgrounds to go with the new background color

<img width="425" alt="Screenshot 2023-10-21 at 18 25 50" src="https://github.com/darklang/darklang.com/assets/87861924/2eed7eea-b95f-434d-ae52-1202487b53e9">

<img width="1057" alt="Screenshot 2023-10-21 at 18 03 12" src="https://github.com/darklang/darklang.com/assets/87861924/788e4fce-6373-46e3-807d-62d0e2d65498">